bin/update-translations.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-
+/* eslint-disable */
```

Comment:
This might be better in the ignore? but not a blocker.

Usually if you are just disabling eslint on a whole file its better in the ignore.